pandas/core/groupby/generic.py

```diff
@@ -1247,14 +1275,16 @@ def _wrap_applied_output(self, keys, values, not_indexed_same=False):
             # make Nones an empty object
             if first_not_none is None:
                 return self.obj._constructor()
-            elif isinstance(first_not_none, NDFrame):
+            elif isinstance(first_not_none, (Series, DataFrame)):
```

Comment:
This change was for mypy to avoid

```
pandas/core/groupby/generic.py:1291: error: Incompatible types in assignment (expression has type "NDFrame", variable has type "Union[DataFrame, Series]")
```

I don't understand it, but it's hopefully harmless.

Reply:
I think this is OK